test/dynamo/test_base_hop.py

```diff
@@ -73,6 +74,88 @@ def forward(self, l_x_: "f32[3, 3]", l_y_: "f32[3, 3]"):
 """,  # NOQA: B950
         )
 
+    def _find_hop_schema(
+        self, gm: torch.fx.GraphModule, target: Any
+    ) -> list[torch._C.FunctionSchema]:
+        import torch.utils._pytree as pytree
+
+        schemas = []
+        for node in gm.graph.find_nodes(op="call_function", target=target):
+
+            def _get_example_value(node: torch.fx.Node) -> Any:
+                if node.op == "get_attr":
+                    return getattr(gm, node.target)
+                else:
+                    return node.meta["example_value"]
+
+            fake_args, fake_kwargs = pytree.tree_map_only(
+                torch.fx.Node,
+                _get_example_value,
+                (node.args, node.kwargs),
+            )
+            schema = node.target.gen_schema(*fake_args, **fake_kwargs)
```

Comment:
Is there a way to include the word "imputed" somehow in the gen_schema API? gen_imputed_schema? I've been using "Imputed schema for HOPs" to refer to this project

Reply:
Hmm maybe gen_schema is better, it's shorter